requirements/base.in

```diff
@@ -0,0 +1,6 @@
+pylint==2.6.0
+pylint-django==2.3.0
```

Comment:
Feels like `Django` would be a base requirement if `pylint-django` is. Not sure how far you want to go with this PR, but we usually let tox manage the Django versions for testing.
See:
* https://github.com/edx/edx-cookiecutters/blob/3079f93e1f72a7d5339e2340fd4a13e49af2c4eb/cookiecutter-django-app/%7B%7Bcookiecutter.repo_name%7D%7D/Makefile#L46-L48
* Also, search for "django" in https://github.com/edx/edx-cookiecutters/blob/3079f93e1f72a7d5339e2340fd4a13e49af2c4eb/cookiecutter-django-app/%7B%7Bcookiecutter.repo_name%7D%7D/tox.ini

Reply:
Again, I tried to be backward-compatible here, so I did not introduce any new requirement.

Reply:
That makes sense. I tend to want to leave notes to my future self around things like this, but I’ll leave that up to you.